tests/test_absoexp.py

```
@@ -3,20 +1,18 @@
 from quantmsio.core.ae import AbsoluteExpressionHander
-
 from .common import datafile
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)